src/components/CartAside/CartAside.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useRef } from 'react';
+import { connect } from 'react-redux';
+
+import { CartAsideItem } from './CartAsideItem';
+import './CartAside.scss';
+
+const CartAside = ({ cartItems }) => {
+
+   const [cartStatus, setCartStatus] = useState(false);
```

Comment:
What does `cartStatus` mean????
`cartStatus == true` ? is item in cart? is cart available? is cart full?
`cartStatus == false` is cart empty?